database/schema.sql

```diff
@@ -0,0 +1,40 @@
+DROP DATABASE IF EXISTS reviews;
+CREATE DATABASE reviews;
+
+\c reviews
+
+CREATE TABLE IF NOT EXISTS list_of_reviews (
+  reviews_id SERIAL PRIMARY KEY,
+  average_rating INTEGER,
+  review_count INTEGER,
+  ratings_id INTEGER FOREIGN KEY REFERENCES review_ratings (rating_id),
+);
+
+CREATE TABLE IF NOT EXISTS review_ratings (
+  ratings_id SERIAL PRiMARY KEY,
+  cleanliness INTEGER,
+  accuracy INTEGER,
+  communication INTEGER,
+  location INTEGER,
+  check_in INTEGER,
+  value INTEGER
+)
+
+CREATE TABLE IF NOT EXISTS review_entry (
+  review_id SERIAL PRIMARY KEY,
+  listing_id INTEGER REFERENCES list_of_reviews (reviews_id),
+  user_id INTEGER REFERENCES user_base(user_id),
+  comment VARCHAR(180),
+  comment_timestamp DATE,
+  owner_id INTEGER,
+  owner_comment VARCHAR(180),
+  owner_comment_timestamp DATE,
```

Comment:
One thing I heard from Karl - Dates can be tricky to use. He said he just uses VARCHAR for dates